autoload/sneak.vim

```diff
@@ -189,6 +189,10 @@ func! sneak#to(op, input, inputlen, count, register, repeatmotion, reverse, incl
   let w:sneak_hl_id = matchadd('Sneak',
         \ (s.prefix).(s.match_pattern).(s.search).'\|'.curln_pattern.(s.search))
 
+  if a:inputlen > 1
+      let w:sneak_current_hl_id = matchadd('SneakCurrent', '\%#.\{'.a:inputlen.'}')
```

Comment:
I expected this to not work with `nomagic` due to this help file info:

```
.	(with 'nomagic': \.)				*/.* */\.*
	Matches any single character, but not an end-of-line.
```

But it appears to work anyway when I tried setting `nomagic`.